src/App.jsx

```diff
@@ -14,57 +16,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelectGoods = (good) => {
+    setSelectProduct(good === selectProduct ? null : good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const GoodsList = ({ goodsList }) =>
+    goodsList.map(good => (
+      <tr
+        data-cy="Good"
+        className={classNames({
+          'has-background-success-light': good === selectProduct,
+        })}
+      >
+        <td>
+          <button
+            data-cy={good === selectProduct ? 'RemoveButton' : 'AddButton'}
+            type="button"
+            className={classNames('button', {
+              'is-info': good === selectProduct,
+            })}
+            onClick={() => handleSelectGoods(good)}
+          >
+            {good === selectProduct ? '-' : '+'}
+          </button>
+        </td>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        <td data-cy="GoodTitle" className="is-vcentered">
+          {good}
+        </td>
+      </tr>
+    ));
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {selectProduct ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectProduct} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectProduct(null)}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
```

Comment:
You have `<h1 className="title is-flex is-align-items-center">` in both cases, only the content is different